app/src/test/java/com/invisibleteam/goinvisible/util/IntentMatcher.java

```diff
@@ -0,0 +1,54 @@
+package com.invisibleteam.goinvisible.util;
+
+import android.content.Intent;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import java.util.Set;
+
+public class IntentMatcher {
+
+    public static Matcher<Intent> containsSameData(final Intent resultIntent) {
```

Comment:
I would like to see a JavaDoc with information that extra values ,which have been added explicitly, are not validated.